src/TableFunctions/TableFunctionFuzzJSON.h

```diff
@@ -5,6 +5,9 @@
 #include <Storages/StorageFuzzJSON.h>
 #include <TableFunctions/ITableFunction.h>
 
+#include "config.h"
```

Comment:
Is this file auto-generated by CMake?

Reply:
Yes, this file is generated by CMake from Common/config.h.in (the location of "config.h" is somewhere below the build directory).